PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -28,15 +29,24 @@ struct StoreAppView: View {
                                 isList: isList,
                                 warningSymbol: $warningSymbol,
                                 warningMessage: $warningMessage)
-        .gesture(TapGesture(count: 2).onEnded {
-            if let url = URL(string: app.link) {
-                downloadApp(url, app)
-            }
-        })
-        .simultaneousGesture(TapGesture().onEnded {
-            selected = app
-        })
         .environmentObject(downloadVM)
+        .contextMenu {
+             Button {
+                 if let url = URL(string: app.link) {
+                     let downloader = DownloadApp(url: url,
+                                                  app: app,
+                                                  warning: warningMessage)
+                     if downloadVM.downloading {
+                         downloader.cancel()
```

Comment:
Cancel does not work the first time I try to select it.

Reply:
Known issue, I have no fix right now.
I would love to have some help on it.

Reply:
fixed the issue btw